driver/csiplugin/controllerserver.go

```diff
@@ -1907,7 +2021,7 @@ func (cs *ScaleControllerServer) DeleteVolume(ctx context.Context, req *csi.Dele
 	}
 
 	relPath := ""
-	if volumeIdMembers.StorageClassType == STORAGECLASS_ADVANCED || volumeIdMembers.VolType == FILE_SHALLOWCOPY_VOLUME {
+	if volumeIdMembers.StorageClassType == STORAGECLASS_ADVANCED || volumeIdMembers.StorageClassType == STORAGECLASS_CACHE || volumeIdMembers.VolType == FILE_SHALLOWCOPY_VOLUME {
```

Comment:
This can be clubbed as volumeIdMembers.StorageClassType != STORAGECLASS_CLASSIC ?

Reply:
@hemalathagajendran

Reply:
Done